js/id/ui/raw_tag_editor.js

```diff
@@ -85,7 +85,11 @@ iD.ui.RawTagEditor = function(context) {
         $items.order();
 
         $items.each(function(tag) {
-            var reference = iD.ui.TagReference({key: tag.key, value: tag.value}, context);
+            var reference;
+            if (id[0] === 'r' && tag.key === 'type')
```

Comment:
I'm a bit unsure about inferring the object's type from it's id – feels a bit hacky…